pkg/nuctl/command/import.go

```diff
@@ -550,14 +570,21 @@ func (i *importProjectCommandeer) importProjects(ctx context.Context,
 		}
 
 		// import project
-		if err := i.importProject(ctx, projectImportOptions); err != nil {
-			return errors.Wrap(err, "Failed to import project")
+		if err := i.importProject(ctx, projectImportOptions, projectReport); err != nil {
+			projectReport.SetFailed(&nuctlcommon.FailReport{
+				FailReason: err.Error()})
+			continue
 		}
 
+		projectReport.Success = true
 		i.rootCommandeer.loggerInstance.InfoWithCtx(ctx, "Successfully imported project",
 			"projectNamespace", projectImportConfig.Project.Meta.Namespace,
 			"projectName", projectName)
 	}
+	errorReport := i.report.SprintfError()
+	if errorReport != "" {
```

Comment:
```suggestion
	if errorReport := i.report.SprintfError(); errorReport != "" {
```

Reply:
?

Reply:
@TomerShor it wasn't ready for review yet